modules/network-observability-virtualization-configuration.adoc

```diff
@@ -0,0 +1,120 @@
+// Module included in the following assemblies:
+//
+// * network_observability/configuring-operator.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="network-observability-virtualization-config_{context}"]
+= Configuring virtual machine (VM) secondary network interfaces for Network Observability
+You can observe networking patterns on an OpenShift Virtualization setup by identifying eBPF-enriched network flows coming from VMs that are connected to additional network interfaces.
+// NOTE to tech review: 
+// do we need to mention anything about CNI here? I noted this in our convo, but I'm not sure if its relevant in the context of this procedure. Maybe its a prerequisite?  
+// You can identify network flows coming from virtual machines connected to additional network interfaces that are using the Network Observability CNI plugin.
+
+ 
+.Prerequisites
+* Access to an {product-title} cluster with an additional network interface, such as a secondary interface or an L2 network.
+* Any other prerequisites?
```

Comment:
As same as [SRIOV case](https://docs.openshift.com/container-platform/4.17/observability/network_observability/configuring-operator.html#network-observability-SR-IOV-config_network_observability), we need to have `spec.agent.ebpf.privileged` field to `true`

cc @msherif1234

Reply:
I have this listed as a FlowCollector config: https://github.com/openshift/openshift-docs/pull/83105/files#diff-2e10b29803e092e329b07fcd472f336c98d04f3eed531135e2afb892b7b22db0R111. I could also call it out in the prerequisites though.

Reply:
yes, that's right, we should probably have dedicated menu item for "secondary interfaces" and list common pre-requisites there.

Reply:
Ok I added this. I feel that the intro text for "secondary interfaces" with Network Observability is lacking, and I'm wondering about potentially mixing up SR-IOV VMs and then SR-IOV interfaces. Are those two separate things? I left a //comment to you guys there trying to parse my confusion 🥴  Just trying to better understand the bigger Network Observability/secondary networks picture